gradle/mrjar.gradle

```diff
@@ -27,14 +33,16 @@ tasks.named('compileJava9Java', JavaCompile).configure {
 
 jar {
     into('META-INF/versions/9') {
-        from sourceSets.java9.output
+        from(sourceSets.java9.output) {
+            /*
+             * As discussed above, we compiled all the classes again. But all
+             * that we want to include under META-INF/versions/9 is the
+             * module-info.
+             */
+            include 'module-info.class'
```

Comment:
:+1:
Great comment, it will help us to understand the intention of this build script.